src/network/MessageRouter.hpp

```diff
@@ -4,6 +4,7 @@
 #include <nlohmann/json.hpp>
 #include <Server/WebSocketServer.hpp>
 #include <utility>
+#include <spdlog/spdlog.h>
```

Comment:
```suggestion
```

Reply:
Die Suggestion verstehe ich nicht? Bspw. nutzt sendMessage spdlog aufgrund des templates schon im Header, von daher ist der include definitiv notwendig.

Reply:
ach so, ich dachte das wäre ausversehen, da ich keine andere änderung in diesem file gesehen habe